main/CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ else()
 endif()
 ROOT_EXECUTABLE(rootnb.exe nbmain.cxx LIBRARIES Core)
 
+#---ReadSpeed-------------------------------------------------------------------------------------
+ROOT_EXECUTABLE(rootreadspeed src/readspeed.cxx $<TARGET_OBJECTS:ReadSpeed> LIBRARIES RIO Tree TreePlayer)
```

Comment:
for my education, what does this `$<TARGET_OBJECTS:ReadSpeed>` do?

Reply:
Going to defer on @bellenot on this one as well